src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -0,0 +1,582 @@
+##########################
+#    Overworld Blocks    #
+##########################
+
+tile.natura.clouds.white.name=Wolke
+tile.natura.clouds.white.tooltip=So leicht und flauschig!\nRettet Dich, wenn Du fällst
+tile.natura.clouds.dark.name=Dunkle Wolke
+tile.natura.clouds.dark.tooltip=So leicht und flauschig!\nVerursacht Blitzeinschläge
+tile.natura.clouds.ash.name=Aschewolke
+tile.natura.clouds.ash.tooltip=So leicht und flauschig!\n3x3 Verwandelt sich in Holzkohle
+tile.natura.clouds.sulfur.name=Schwefelwolke
+tile.natura.clouds.sulfur.tooltip=So leicht und flauschig!\n2x2 Verwandelt sich in Schwefel
+
+tile.natura.overworld_logs.maple.name=Ahornholz
+tile.natura.overworld_logs.maple.tooltip=Etwas süßlich
+tile.natura.overworld_logs.silverbell.name=Silberglöckchenholz
```

Comment:
I think this one's name is "Schneeglöckchenbaum".

Reply:
Yeah, seems so, thanks. :)